CONTRIBUTING.md

```diff
@@ -1,145 +1,125 @@
+* [Code of Conduct](#code-of-conduct)
+* [Developing](#developing)
+* [Submitting a new feature](#submitting-a-new-feature)
+* [Reporting bugs](#reporting-bugs)
+* [Contributing](#getting-in-contact)
+* [Coding conventions](#coding-conventions)
 
-### Git Commit Guidelines
+## Code of Conduct
+
+This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT.md). By contributing, you are expected to honor these guidelines.
 
-We follow precise rules for git commit message formatting. These rules make it easier to review commit logs and improve contextual understanding of code changes. This also allows us to auto-generate the CHANGELOG from commit messages.
+## Developing
 
-Each commit message consists of a **header**, **body** and **footer**.
+### Installation
 
-#### Header
+1. Fork the repository `one-app-router` to your GitHub account.
+2. Afterwards run the following commands in your terminal
 
-The header is required and must not exceed 70 characters to ensure it is well-formatted in common git tools. It has a special format that includes a *type*, *scope* and *subject*:
+    ```bash
+    $ git clone https://github.com/<your-github-username>/one-app-router
+    $ cd one-app-router
+    ```
 
-Syntax:
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
+    ```
 
-The *type* should always be lowercase as shown below.
+4. You can now run any of these scripts from the root folder.
 
-##### Allowed `<type>` values:
+#### Running and cleaning the build files
 
-* **feat** (new feature for the user)
-* **fix** (bug fix for the user, not a fix to build scripts)
-* **docs** (changes to documentation)
-* **style** (formatting, missing semi colons, etc; no functional code change)
-* **refactor** (refactoring production code, eg. renaming a variable)
-* **test** (adding missing tests, refactoring tests; no production code change)
-* **chore** (updating build/env/packages, etc; no production code change)
+- `npm run build`
 
-#### Scope
+Runs `babel` to compile `src` files to transpiled JavaScript into `lib` using
+[`babel-preset-amex`](https://github.com/americanexpress/babel-preset-amex).
 
-The *scope* describes the affected code. The descriptor may be a route, component, feature, utility, etc. It should be one word or camelCased, if needed:
+#### Running tests
 
-```bash
-feat(transactions): added column for quantity
-feat(BalanceModule): initial setup
-```
+- `npm run lint`
 
-The commit headers above work well if the commit affects many parts of a larger feature. If changes are more specific, it may be too broad. To better clarify specific scopes, you should use a `feature/scope` syntax:
+Verifies that your code matches the American Express code style defined in
+[`eslint-config-amex`](https://github.com/americanexpress/eslint-config-amex).
```

Comment:
again this is not currently true, as this is not setup with `eslint-config-amex`

Reply:
updated 42cb65fc